src/CMakeLists.txt

```diff
@@ -468,6 +468,7 @@ if(USE_SDL2)
     list(APPEND GAME_COMPILE_OPTIONS -DBUILD_WITH_SDL2)
 
     list(APPEND GAMEENGINE_SRC
+        platform/sdl2/sdl2keybd.cpp
```

Comment:
`sdl2keyboard` is ok name. There is no need for short acronyms.

Reply:
i used that acronym due to `dinputkeybd.cpp`

Reply:
ok